boards/arm/usb_kw24d512/Kconfig.defconfig

```diff
@@ -0,0 +1,82 @@
+# Kconfig - USB-KW24D512 board
+#
+# Copyright (c) 2017, Phytec Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BOARD_USB_KW24D512
+
+config BOARD
+	default usb_kw24d512
+
+config OSC_XTAL0_FREQ
+	default 4000000
```

Comment:
When the modem is initialized like in set_modem_clock, it is 4MHz (Table 5-10, CLOCK_OUT ist 4MHz if GPIO5=0 after reset, GPIO5 is connected to PTC0, CLOCK_OUT to EXTAL0)